chirpstack/src/config.rs

```diff
@@ -422,9 +422,28 @@ pub struct OpenIdConnect {
     pub login_redirect: bool,
     pub login_label: String,
     pub assume_email_verified: bool,
+    pub additional_scopes: Vec<String>,
 }
 
-#[derive(Serialize, Deserialize, Default, Clone)]
+impl Default for OpenIdConnect {
+    fn default() -> Self {
+        OpenIdConnect {
+            registration_enabled: false,
+            registration_callback_url: "".to_string(),
+            provider_url: "".to_string(),
+            client_id: "".to_string(),
+            client_secret: "".to_string(),
+            redirect_url: "".to_string(),
+            logout_url: "".to_string(),
+            login_redirect: false,
+            login_label: "".to_string(),
+            assume_email_verified: false,
+            additional_scopes: vec![],
```

Comment:
E.g. here we would define `scopes: vec!["email".to_string(), "profile".to_string()],`